modules/process-testing/pages/task.adoc

```diff
@@ -52,15 +52,15 @@ class ProcessIT {
         userTask.execute(walter, contract);
 
         // Use the provided predicates to ensure that a taks has been correctly executed before to continue.
-        await().until(userTask, hasBeenExecutedBy(walter) <7>
+        await("The task 'aTask' has been completed").until(userTask, hasBeenExecutedBy(walter) <7>
                 .and(taskArchived())
                 .and(taskCompleted()));
     }
 }
 ----
 <1> Retrieve the pending task _aTask_ from the process instance.
 <2> Create a contract from a dedicated method to execute the task (more details about contract creation xref:contract.adoc[here]).
-<3> Retrieve the candidates of the task (i.e the users that can be assigned to the task), and ensure that walter is the only candidate.
+<3> Retrieve the candidates of the task (i.e the users that can be assigned to the task), and ensure that walter is the only candidate (the predicate `hasCandidates(String... candidates)` can be used to check the candidates in case there is an actor filter that takes time).
```

Comment:
```suggestion
<3> Retrieve the candidates of the task (i.e the users that can be assigned to the task), and ensure that walter is the only candidate (the predicate `hasCandidates(User... candidates)` can be used to check the candidates in case there is an actor filter that takes time).
```